services/secondary-storage/server/src/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java

```diff
@@ -942,6 +931,83 @@ protected Answer copyFromNfsToS3(CopyCommand cmd) {
         }
     }
 
+    protected boolean swiftUploadMetadataFile(SwiftTO swift, File srcFile, String containerName) throws IOException {
+
+
+        //create a template.properties for Swift with its correct unique name
```

Comment:
Since you went through the trouble of cleaning up some of the code, how about you improve it some more by turning this comment into a javadoc comment for this method, also describing the parameters involved. 

If you like this idea, I'd suggest going ahead and writing another javadoc comment for the `copyFromNfsToSwift` method.

Little things like these can go a long way to improve code readability. :)